### PR TITLE
Create ui-c8y-1021-0-0-add-message-banner-to-branding-editor-7085.md

### PR DESCRIPTION
**Pull request by gen-App-Enablement**
Release note created by jdre-c8y

Ticket: [MTM-61123](https://cumulocity.atlassian.net/browse/MTM-61123)
Original PR: [7085](https://github.com/Cumulocity-IoT/cumulocity-ui/pull/7085)

[MTM-61123]: https://cumulocity.atlassian.net/browse/MTM-61123?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: content/change-logs/application-enablement/ui-c8y-1021-0-0-add-message-banner-to-branding-editor-7085.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
+title: Add message banner to branding editor (#7085)
```
Comment (gen-App-Enablement) [inline content/change-logs/application-enablement/ui-c8y-1021-0-0-add-message-banner-to-branding-editor-7085.md]: The Claude AI suggested to use the following title:
```suggestion
title: Message banner added to branding editor
```

File: content/change-logs/application-enablement/ui-c8y-1021-0-0-add-message-banner-to-branding-editor-7085.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: component-YbYJ3gLU_
+    label: Web SDK
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: MTM-61123
+version: 1021.0.0
+---
+Add message banner to branding editor (#7085)
```
Comment (gen-App-Enablement) [inline content/change-logs/application-enablement/ui-c8y-1021-0-0-add-message-banner-to-branding-editor-7085.md]: The Claude AI suggested to use the following body:
```suggestion
To provide a more flexible and customizable user experience, a new message banner feature has been added to the branding editor. This change allows administrators to create and display custom messages or announcements within their branded applications. The message banner can be used to inform users about important updates, maintenance schedules, or any other relevant information. This feature enhances the communication capabilities of the platform and enables administrators to effectively convey important messages to their users without the need for additional tools or workarounds.
```

**Review by gen-App-Enablement — COMMENTED** [review]
AI improvements

File: content/change-logs/application-enablement/ui-c8y-1021-0-0-add-message-banner-to-branding-editor-7085.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: component-YbYJ3gLU_
+    label: Web SDK
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: MTM-61123
+version: 1021.0.0
+---
+To provide a more flexible and customizable user experience, a new message banner feature has been added to the branding editor. This change allows administrators to create and display custom messages or announcements within their branded applications. The message banner can be used to inform users about important updates, maintenance schedules, or any other relevant information. This feature enhances the communication capabilities of the platform and enables administrators to effectively convey important messages to their users without the need for additional tools or workarounds.
```
Comment (BeateRixen) [inline content/change-logs/application-enablement/ui-c8y-1021-0-0-add-message-banner-to-branding-editor-7085.md]:
```suggestion
To provide a more flexible and customizable user experience, a new message banner option has been added to the branding editor. This feature allows administrators to create and display custom messages or announcements within their branded applications. The message banner can be used to inform users about important updates, maintenance schedules, or any other relevant information. This feature enhances the communication capabilities of the platform and enables administrators to effectively convey important messages to their users without the need for additional tools or workarounds.
```

**Review by BeateRixen — APPROVED** [review]

**Comment by jdre-c8y** [comment]
@rahultsag  can you have a look?

**Review by rahulc8y — APPROVED** [review]

**Pull request merged**